20150310-wp-core-contrib-day.md

```diff
@@ -0,0 +1,255 @@
+
+As you make your changes, use `grunt` to refresh the `build` directory so that you can test your changes as you go along.
+
+If at all possible, please add unit tests or update the unit tests in the `tests` directory that help ensure your code integrates well. Be sure to [Run The Unit Tests][] every now and then, and most definitely when you think you are finished with your development and testing and are ready to create a patch.
+
+### Creating a Patch
+You've developed and tested your WordPress Core changes and are now ready to pass your changes back to the WordPress Core team for reviewing and testing, so how do you do that?
+
+It so happens that if you run the following, you'll get exactly what is needed to patch another checkout of the repository:
+
+```
+svn diff
+```
+
+![wordpress-core — Creating a Patch](https://s3.amazonaws.com/cdn.deliciousbrains.com/content/uploads/2015/03/wordpress-core-—-Creating-a-Patch.png)
+
+So, to create your patch file simply redirect the output of `svn diff` to a file:
+
+```
+svn diff > 14584.4.patch
+```
+
+Notice that in the example we're creating a file with an increased version number, even though on the surface nothing has changed in this example, we have generated this from a much later version of the repository, and so should increase the version number if we're going to the trouble of refreshing and re-submitting the patch for the ticket.
+
+### Submit The Patch
+Submitting the patch file is relatively simple. Make sure you're logged in to wordpress.org, navigate to the ticket in question and you should find an Attachments block nestled between the main description block and the Change History.
+
+![Attachments Block In a Ticket](https://s3.amazonaws.com/cdn.deliciousbrains.com/content/uploads/2015/03/Attachments-Block-In-a-Ticket.png)
+
+In the Attachments block there is an "Attach file" button which leads you to a page where you can choose your file to upload and give a brief description.
+
+![Submitting a Ticket Attachment](https://s3.amazonaws.com/cdn.deliciousbrains.com/content/uploads/2015/03/Submitting-a-Ticket-Attachment.png)
+
+![Ticket Attachment Submitted](https://s3.amazonaws.com/cdn.deliciousbrains.com/content/uploads/2015/03/Ticket-Attachment-Submitted.png)
+
+### Modify The Ticket
+Near the bottom of the WordPress Core ticket you've been working on, you should see a "Modify Ticket" block.
+
+![Modify Ticket Block](https://s3.amazonaws.com/cdn.deliciousbrains.com/content/uploads/2015/03/Modify-Ticket-Block.png)
+
+Depending on what you have done or found, you may need to alter some of the Workflow Keywords within the **Change Properties** section.
```

Comment:
Need to be consistent with formatting of section titles, previously not made Attachments bold